bowlingScore.test.js

```diff
@@ -0,0 +1,42 @@
+const BowlingGame = require('./bowlingScore')
+
+describe('bowlingGame', () => {
+  it('makes a bowling class', () => {
+    const bowlingGame = new BowlingGame();
+  })
+  it('han an empty array for frames', () => {
+    const bowlingGame = new BowlingGame();
```

Comment:
Using `beforeEach()` , you can set up the `new BowlingGame()` object just once at the top of your code. A bit like using subject in Ruby.